src/objectExplorer/objectExplorerFilter.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { ReactWebViewPanelController } from "../controllers/reactWebviewController";
+import * as vscodeMssql from 'vscode-mssql';
+import * as vscode from 'vscode';
+import { WebviewRoute } from "../sharedInterfaces/webviewRoutes";
+import { TreeNodeInfo } from "./treeNodeInfo";
+import { ObjectExplorerFilterState, ObjectExplorerReducers } from "../sharedInterfaces/objectExplorerFilter";
+
+export class ObjectExplorerFilterReactWebviewController extends ReactWebViewPanelController<ObjectExplorerFilterState, ObjectExplorerReducers> {
```

Comment:
I wonder if we really care that this is a "ReactWebview" controller?  Isn't the fact that it uses react kinda an implementation details, or is the controller so coupled with React that it wouldn't make sense to use the framework without it?  Basically, should we ditch React in all the identifiers and just call it the `WebViewContoller`?

Reply:
There was already a webview controller. I added react prefix to this in order to separate from the old code. Once we get rid of those I can rename this class